packages/rum/src/types.ts

```diff
@@ -129,3 +129,12 @@ export interface InternalContext {
     id: string
   }
 }
+
+export type ManuallyAddedErrorSource = ErrorSource.CUSTOM | ErrorSource.SOURCE | ErrorSource.NETWORK
```

Comment:
from a user perspective it is a mouthful, it could be nicer to either:
- rename it to ErrorSource and prefix the core one with internal
- rename it to RumErrorSource

Reply:
Since the ErrorSource is used by the RumEvent type exported publicly, I don't think we should rename it "internal" (it is user facing). Also, adding the `Rum` prefix would be confusing, because we would have two user facing types with a slightly different name.

Instead of having a type for this, we could simply change the addError signature to list the possible source values like:

```ts
addError(error: unkwown, context?: Context, source: ErrorSource.CUSTOM | ErrorSource.SOURCE | ErrorSource.NETWORK)
```

WDYT?

Reply:
LGTM